genai-perf/genai_perf/config/input/create_config.py

```diff
@@ -95,11 +96,16 @@ def _add_cli_options_to_config(
 
         CreateConfig._check_that_override_is_set(args)
         CreateConfig._add_top_level_args_to_config(config, args)
+        CreateConfig._add_output_args_to_config(config, args)
```

Comment:
This isn't a clean way to do this as you are relying on an implicit ordering of the function calls. A better way is to make this:
```
if config.subcommand == Subcommand.PROCESS:
   CreateConfig._add_process_args_to_config()
else:
   CreateConfig._add_profile_analyze_args_to_config()
```
And then call the exact args_to_config methods you need from these new methods